platform/ui/tailwind.config.js

```diff
@@ -278,10 +271,10 @@ module.exports = {
       ],
     },
     fontSize: {
-      xs: '0.75rem',
-      sm: '0.875rem',
-      base: '1rem',
-      lg: '1.125rem',
+      xs: '0.65rem',
+      sm: '0.75rem',
+      base: '0.875rem',
+      lg: '1rem',
```

Comment:
Were these changes made to align with Dan's design?
Better to bump these, or because these are relative to font-size of the root element, bump the font-size there?